packages/terra-table/src/TableHeaderCell.jsx

```diff
@@ -25,49 +26,72 @@ const propTypes = {
   /**
    * Content to be displayed for the column header
    */
-  content: PropTypes.node.isRequired,
+  children: PropTypes.node,
   /**
-   * The minimum width for the column
+   * Whether or not row is selectable
```

Comment:
Should this documentation say that the row is the parentRole?

Reply:
Updated.